.travis.yml

```diff
@@ -4,7 +4,15 @@ sudo: required
 dist: xenial
 dotnet: 2.2
 
+addons:
+ apt:
+  sources:
+   - sourceline: 'ppa:mike42/mdcheckr'
+  packages:
+   - mdcheckr
+
 script:
+ - git ls-files | grep '\.md$' | tr '\n' '\0' | xargs -0 mdcheckr
```

Comment:
Nice find, making sure that the build breaks if a link is broken! :+1:

Reply:
https://github.com/mike42/mdcheckr, for reference

Reply:
Ah, neat; it detects other syntax/parse errors as well!